DotNet/Shared/Application/Models/KafkaTopic.cs

```diff
@@ -68,5 +68,6 @@ public enum KafkaTopic
     ReadyForValidation,
     ValidationComplete,
     [StringValue("ValidationComplete-Retry")]
-    ValidationCompleteRetry
+    ValidationCompleteRetry,
+    SubmitPayLoad
```

Comment:
Seeing mixed casing (`Payload` vs. `PayLoad`) in a few different places.  Probably not an issue elsewhere if binding is case-insensitive.  This instance might be problematic, though.  (Will a producer/consumer fail if it disagrees with the broker on topic name casing?)  What I've seen in Jira and Confluence is `Payload`, so recommend standardizing on that.  Also need to add the topic name to `topics.txt` in the root of the repo.